### ISSUE TITLE
After fresh installation of SuiteCRM is sidebar empty

### ISSUE THREAD

**Issue by Mausino** [salesagility/SuiteCRM#2442]
#### Issue

I am thinking that after fresh instal of SuiteCRM should not show empty sidebar. For me it looks very wird.

![sidebar](https://cloud.githubusercontent.com/assets/5773777/19489254/c8f217a4-956a-11e6-8439-5a76c30746e2.png)
#### Expected Behavior

Will show the Recently Viewed and Favorites (headers)
#### Actual Behavior

Is nothing there. 
#### Your Environment

- SuiteCRM Version used:
- Browser name and version (e.g. Chrome Version 51.0.2704.63 (64-bit)):
- Environment name and version (e.g. MySQL, PHP 7):
- Operating System and version (e.g Ubuntu 16.04):

**Comment by shogunpol** [comment]
@Mausino , yes it happen it is expected behaviour of the system, this is by design.
If there is no recent records or favourites the headers just wont fires.
I suggest close this issue, if not we can label it as a suggestion.
Thanks for your contribution.

**Comment by Mausino** [comment]
@shogunpol can you mark it as suggestion? Thanks

**Comment by samus-aran** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [After fresh installation of SuiteCRM is sidebar empty](https://trello.com/c/7aikbY0E/64-after-fresh-installation-of-suitecrm-is-sidebar-empty)

**Comment by samus-aran** [comment]
We have now set up a new home for suggestions at Trello.  All github issues that were labeled 'suggestion' have been moved and will be closed.  Certain ones will be progressed within the new Suggestion Box and may be re-opened.

Announcement of moving Suggestions:
https://suitecrm.com/forum/suggestion-box/13691-moving-suggestions-from-github-to

New SuiteCRM Suggestion Box
https://trello.com/b/Ht7LbMqw/suitecrm-suggestion-box
